lib/rules/syntaxes/slot-attribute.js

```diff
@@ -95,6 +101,9 @@ module.exports = {
      * @returns {void}
      */
     function reportSlot(slotAttr) {
+      if (ignore.includes(slotAttr.parent.parent.name)) {
```

Comment:
Please also ignore other casings:

```ts
const casing = require('../utils/casing')

const componentName = slotAttr.parent.parent.name
if (
  ignore.has(componentName) ||
  ignore.has(casing.pascalCase(componentName)) ||
  ignore.has(casing.kebabCase(componentName))
) { … }
```

And please add test cases for that.

Reply:
Fixed: [847a098](https://github.com/vuejs/eslint-plugin-vue/pull/2314/commits/847a0986d974e78ed66f21ce03857365a261b329)